snapper/AsciiFile.h

```diff
@@ -80,13 +82,14 @@ namespace snapper
 
     protected:
 
-	const string Name_C;
-	const bool remove_empty;
-
 	vector<string> Lines_C;
 
-    };
+    private:
 
+	string Name_C;
+	bool remove_empty;
```

Comment:
Maybe it does follow some style guide but not one I follow. It is just old code.